src/content/keyAutoAdd/testWallet.ts

```diff
@@ -88,16 +85,13 @@ const getAccountDetails: Step<never, Account[]> = async ({
  * with a different account (user disconnected and changed account), revoke from
  * there first.
  */
-const revokeExistingKey: Step<typeof getAccountDetails, Account[]> = async (
-  { keyId, skip },
-  accounts,
-) => {
+const revokeExistingKey: Run<void> = async ({ keyId }, { skip, output }) => {
+  const accounts = output(getAccounts);
```

Comment:
That's the main thing.

Reply:
The comment for this function is confusing me a bit. If the user disconnected and changed the account, we wouldn't be able to see the key in the keys page.

Reply:
Say, user connected once to `USD -> Addr#1`. Then disconnected. The key is still there in wallet added to `USD -> Addr#1` account.
If now I want to connect my `EUR -> Addr#1` account, the extension still has the same key. So adding it will throw an `internal server error`. That's why we first revoke existing key (by ID) if any (from any existing account/address).

Reply:
Oh, I thought you were referring to a Test Wallet account in the comment. Ok, makes sense now. Let's add a TODO for this since we will probably won't have to do this in the future: users should be able to upload the same key to multiple wallet addresses, but not to the same one.